src/descriptor_runner/webdnn.ts

```diff
@@ -191,6 +192,18 @@ export async function load(directory: string, initOption: InitOption = {}): Prom
     backendOrder = backendOrder.slice();
     if (backendOrder.indexOf('fallback') === -1) backendOrder.concat(['fallback']);
 
+    // FIXME: User-defined transformDelegate is overridden
```

Comment:
I noticed that the function `registerTransformUrlDelegate` is not exported by module `webdnn`, so actually users cannot use this function as the document said (https://mil-tokyo.github.io/webdnn/docs/api_reference/descriptor-runner/modules/webdnn.html#registertransformurldelegate).  I think it is better to make `transformUrlDelegate` be as an init option, instead of exporting `registerTransformUrlDelegate` that allows user to modify `transformUrlDelegate` globally. Do you think it's a good idea? I'll push some code about it later.